app/models/planning_application.rb

```diff
@@ -1029,7 +1029,11 @@ def set_key_dates
   end
 
   def application_type_determination_period
-    application_type.determination_period_days || DAYS_TO_EXPIRE
+    if pre_application?
+      local_authority.application_type_overrides.determination_period_days
+    else
+      application_type.determination_period_days || DAYS_TO_EXPIRE
+    end
```

Comment:
What happens here if the LA hasn't configured a determination period? Can we have it all back to the default if one isn't set?

Reply:
Good spot, I had this logic but looks like it might be change when I was rebasing.